migrate.js

```diff
@@ -0,0 +1,42 @@
+const taskcluster = require('taskcluster-client');
+const auth = new taskcluster.Auth();
+
+const migrate = async () => {
+  try {
+    const roles = await auth.listRoles();
+
+    // Migrate mozillians-unvouched
+    await auth.createRole('everybody', {
+      scopes: await auth.role('mozillians-unvouched').scopes,
+      description: 'Role assigned to everybody. It should only have the scopes required to run the tutorial, and nothing that might harm other users of Taskcluster.',
+    });
+
+    roles
+      .filter(({ roleId }) => roleId.startsWith('mozilla-user:'))
+      .forEach(async ({roleId, scopes}) => {
```

Comment:
I think you want to `await .map(async ...)` here, otherwise you'll have unhandled rejections.

It might be better to do this sequentially (I know, cool kids don't use `for` loops anymore..).  The auth service is going to process them that way anyway.